validphys2/src/validphys/config.py

```diff
@@ -1271,12 +1278,15 @@ def parse_default_filter_rules_recorded_spec_(self, spec):
         return spec
 
     def parse_added_filter_rules(self, rules: (list, type(None)) = None):
-        return rules
+        """
+        Returns a tuple of AddedFilterRule objects. The rules are frozen dataclasses.
+        AddedFilterRule objects inherit from FilterRule objects.
```

Comment:
Why is `AddedFilterRule` not just a `FilterRule`?

Reply:
They are the same indeed (Added... inherits from FilterRule). But it might be useful to differentiate between the two for instance for error messages or don't know what.

So I think I prefer having two "different" objects for these

Reply:
> What is the purpose of this class?
Hi @RoyStegeman  there was a little discussion here that might answer your question.

Reply:
Okay thanks, I wasn't sure it was intentional or forgotten